src/cs/lib/msquic_generated.cs

```diff
@@ -2580,6 +2591,9 @@ internal static unsafe partial class MsQuic
         [NativeTypeName("#define QUIC_PARAM_CONFIGURATION_VERSION_SETTINGS 0x03000002")]
         internal const uint QUIC_PARAM_CONFIGURATION_VERSION_SETTINGS = 0x03000002;
 
+        [NativeTypeName("#define QUIC_PARAM_CONFIGURATION_SCHANNEL_CREDENTIAL_ATTRIBUTE_W 0x03000003")]
+        internal const uint QUIC_PARAM_CONFIGURATION_SCHANNEL_CREDENTIAL_ATTRIBUTE_W = 0x03000003;
```

Comment:
I just realized, both the Rust header and documentation needs to be updated too. For the recent changes as well.

Reply:
Is that a manual or automated process?

Reply:
I'll do all three of them in a separate PR

Reply:
Ok.